DTML.EduBot/Dialogs/LessonDialog.cs

```diff
@@ -53,22 +55,10 @@ public LessonDialog(Lesson lesson)
                     new Image()
                     {
                         Size = ImageSize.Large,
-                        Url  = "http://i1.wp.com/www.foodrepublic.com/wp-content/uploads/2012/03/roston_rotteneggs.jpg"
+                        Url  = nextTopic.ImageUrl
```

Comment:
If a card doesn't have an image? Would it show the card without image? or does it show a blank image?